config/io.github.insilmaril.vym.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>io.github.insilmaril.vym</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LGPL-2.1-or-later</project_license>
+    <name>vym</name>
+    <summary>Tool for generating and manipulating mind maps</summary>
+    <description>
+	<p>
+	    VYM (View Your Mind) allows you to easily and intuitively
+            organize your ideas in a structured way. You can create 
+            metal maps and export them to multiple formats.
+	</p>
+    </description>
+    <screenshots>
+        <screenshot type="default">
+          <image>https://a.fsdn.com/con/app/proj/vym/screenshots/328319.jpg</image>
```

Comment:
Meanwhile we could have nicer screenshots, but I guess these still can be updated later ;-)

Reply:
The package has already been incorporated into flathub, the changes will have to be made in the next update.